llvm/test/MC/AMDGPU/gfx12_asm_opsel.s

```diff
@@ -0,0 +1,48 @@
+// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize32,+real-true16 -show-encoding %s | FileCheck --check-prefixes=GFX12 %s
```

Comment:
It looks like in gfx11, we have moved the tests with opsel from gfx11_asm_vop3.s to gfx11_asm_opsel.s. That seems ok to me. In gfx12 though, we still have the op_sel tests in gfx12_asm_vop3.s. So perhaps this test file is not needed?

Reply:
Should I move remove op_sel from gfx12_asm_vop3.s just like in gfx11?

Reply:
Talked offline. I will remove this file first since it save me efforts for the upstreaming. I can move the gfx12 op_sel test to this seperate file later after all upstreaming has been done